config/config-builder/src/validator_config.rs

```diff
@@ -71,6 +75,11 @@ impl ValidatorConfig {
         self
     }
 
+    pub fn build_waypoint(&mut self, enabled: bool) -> &mut Self {
```

Comment:
why not just making this field public to avoid the setter

Reply:
because that is the way, I foolishly, wrote the entire config-builder app.

Reply:
go back to java :P